src/evo/cbtx.h

```diff
@@ -62,6 +68,7 @@ class CCbTx
             if (nVersion >= CB_V20_VERSION) {
                 obj.pushKV("bestCLHeightDiff", static_cast<int>(bestCLHeightDiff));
                 obj.pushKV("bestCLSignature", bestCLSignature.ToString());
+                obj.pushKV("assetLockedAmount", ValueFromAmount(assetLockedAmount));
```

Comment:
@knst @PastaPastaPasta @ogabrielides @UdjinM6 A little late to the party here, but what do you think about calling it smth like `creditPoolBalance` instead? It wasn't clear to me at first if "assetLockedAmount" referred just to the asset locks in this block or the system-wide asset lock balance.

Reply:
I agree about `creditPoolBalance` - maybe that's not perfect name still, but definitely better than assetLockedAmount.

Let's gather feedback from everyone and I will prepare PR to change it.

Reply:
Yup, I like `creditPoolBalance `

Reply:
makes sense

Reply:
Here it does: https://github.com/dashpay/dash/pull/5526